tests/drivers/adc/adc_api/src/test_adc.c

```diff
@@ -121,6 +121,78 @@
 #define ADC_1ST_CHANNEL_ID	3
 #define ADC_2ND_CHANNEL_ID	4
 
+#elif defined(CONFIG_BOARD_NUCLEO_L073RZ)
+#define ADC_DEVICE_NAME		DT_ADC_1_NAME
+#define ADC_RESOLUTION		10
+#define ADC_GAIN		ADC_GAIN_1
+#define ADC_REFERENCE		ADC_REF_INTERNAL
+#define ADC_ACQUISITION_TIME	ADC_ACQ_TIME_DEFAULT
+#define ADC_1ST_CHANNEL_ID	0
+
+#elif defined(CONFIG_BOARD_NUCLEO_F091RC)
+#define ADC_DEVICE_NAME		DT_ADC_1_NAME
+#define ADC_RESOLUTION		12
+#define ADC_GAIN		ADC_GAIN_1
+#define ADC_REFERENCE		ADC_REF_INTERNAL
+#define ADC_ACQUISITION_TIME	ADC_ACQ_TIME_DEFAULT
+#define ADC_1ST_CHANNEL_ID	0
+
+#elif defined(CONFIG_BOARD_NUCLEO_F103RB)
+#define ADC_DEVICE_NAME		DT_ADC_1_NAME
+#define ADC_RESOLUTION		12
+#define ADC_GAIN		ADC_GAIN_1
+#define ADC_REFERENCE		ADC_REF_INTERNAL
+#define ADC_ACQUISITION_TIME	ADC_ACQ_TIME_DEFAULT
+#define ADC_1ST_CHANNEL_ID	0
```

Comment:
These parameters are exactly the same as the ones for `NUCLEO_F091RC`. Wouldn't it be better to use:
```
#elif defined(CONFIG_BOARD_NUCLEO_F091RC) || \
      defined(CONFIG_BOARD_NUCLEO_F103RB)
```

Same applies to a couple of Nucleo boards using 10-bit resolution.

Reply:
Yes, that's better